src/renderers/webgl/WebGLBackground.js

```diff
@@ -68,7 +68,7 @@ function WebGLBackground( renderer, cubemaps, cubeuvmaps, state, objects, alpha,
 			if ( boxMesh === undefined ) {
 
 				boxMesh = new Mesh(
-					new BoxGeometry( 1, 1, 1 ),
+					new BoxGeometry( 10000, 10000, 10000 ),
```

Comment:
Is this change necessary?

Reply:
According to @snagy: the WebGLBackground code has some special casing and is not multiview-aware, the original dimension of 1x1x1 becomes really obvious as a box around your head with multiview, and making the box really big effectively negates this effect.

Reply:
I see. concern is that this might affect non-vr (2D) applications perhaps causing the background to be clipped? 10,000 is 10km